checks/http/check.go

```diff
@@ -0,0 +1,65 @@
+package http
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"time"
+)
+
+// Config is the HTTP checker configuration settings container.
+type Config struct {
+	// URL is the remote service health check URL.
+	URL string
+	// RequestTimeout is the duration that health check will try to consume published test message.
+	// If not set - 5 seconds
+	RequestTimeout time.Duration
+	// LogFunc is the callback function for errors logging during check.
+	// If not set logging is skipped.
+	LogFunc func(err error, details string, extra ...interface{})
+}
+
+// New creates new HTTP service health check that verifies the following:
+// - connection establishing
+// - getting response status from defined URL
+// - verifying that status code is less than 500
+func New(config Config) func() error {
+	return func() error {
+		if config.LogFunc == nil {
+			config.LogFunc = func(err error, details string, extra ...interface{}) {}
+		}
+
+		if config.RequestTimeout == 0 {
+			config.RequestTimeout = time.Second * 5
+		}
+
+		req, err := http.NewRequest(http.MethodGet, config.URL, nil)
```

Comment:
no circuit-breakers ? :trollface:

Reply:
Are we going to check service status or circuit breaker status? =)